corona-warn-app/cwa-app-android
# QR Code Poster (EXPOSUREAPP-5962)

- Create location poster PDF
- Enable printing , Sharing
- Delete testing screens, Please use the actual screens 

### Testing
- Test against mock server
- Create location -> Choose Print option from overflow menu
- Create location -> QR Code details screen -> Print button
- Check poster and try printing and sharing 

### TODO
- [x] Finalise positioning values from server
- Text Box font - not provided yet
- Create poster on the fly (New ticket 🎫 probably or even after MVP)

After merging https://github.com/corona-warn-app/cwa-app-android/pull/2737
- [x] Use Location Url to generate QR Code 
- [x] Extend testability for TraceLocation
